german/bookmark/agot_bm_group_AV_l_german.yml

```diff
@@ -1,14 +1,14 @@
\ No newline at end of file
+l_german:
+ agot_bm_group_AV:0 "Die Invasion der Andalen\n6699 v.A.E. - 6478 v.A.E."
+ agot_bm_6478:0 "Die Geißelung von Lorath"
+ agot_bm_6478_desc:0 "Nachdem er seine Herrschaft über Andalos gefestigt hat, hat Hochkönig Qarlon der Große die Eroberung der Freien Stadt Norvos im Visier."
+ agot_bm_6538:0 "Die Schlacht der Sieben Sterne"
+ agot_bm_6538_desc:0 "Unter der Führung von König Robar Royce versammeln sich die Ersten Menschen des Tals, um sich ein letztes Mal gegen die andalischen Eindringlinge zu wehren. Auch die Andalen haben sich hinter Ser Artys Arryn, einem jungen Krieger, der als Falkenritter bekannt ist, versammelt."
+ agot_bm_6583:0 "Der Fall der Graueisen"
+ agot_bm_6583_desc:0 "Tausend Jahre nach ihrer Landung auf den Fingern haben die Andalen ihre Aufmerksamkeit auf die Eiseninseln gelenkt. Als die Andalen in die Flusslande, die Westlande und die Weite eindrangen und sich dort niederließen, begannen sie mit dem Bau neuer Siedlungen und stabiler Festungen. Diese stärkeren Verteidigungsanlagen in Verbindung mit der Segelfertigkeit der Andalen haben den Einfluss der Eisengeborenen auf das Meer der Abenddämmerung geschwächt. Die Andalen begannen, in Wellen in die Eiseninseln einzufallen, oft mit Unterstützung rivalisierender eisengeborener Lords. Nun muss sich Rognar II. Graueisen, König der Eiseninseln, einem Bündnis der Andalen mit den Häusern Drumm, Graufreud, Hoffartt und Orkwald stellen."
+ agot_bm_6610:0 "Der Hungrige Wolf"
+ agot_bm_6610_desc:0 "König Theon Stark regiert den Norden zu einer Zeit, in der er von allen Seiten bedrängt wird. Die Eisengeborenen überfallen die Westküste, die durch ihre Siege im Süden ermutigten Andalen sind bekannt dafür, an der Ostküste zu landen, und die Bedrohung durch die Wildlinge jenseits der Mauer ist nicht zu unterschätzen. Anderswo in Westeros kämpfen die Ersten Menschen weiterhin gegen die scheinbar endlose Flut andalischer Invasoren."
+ agot_bm_6650:0 "Der Hammer der Gerechtigkeit"
+ agot_bm_6650_desc:0 "Während die Andalen weiterhin in Westeros einfallen, etabliert sich König Tristan Schlamm aus den Flusslanden als größter Verteidiger der Ersten Menschen. Er ist als Hammer der Gerechtigkeit bekannt und hat 99 Schlachten gegen die Invasoren gewonnen. Nachdem er kürzlich eine Invasion von König Roland Arryn zurückgeschlagen hat, sieht sich der Hammer der Gerechtigkeit nun dem mächtigen andalischen Kriegsherrn Eremitans Vanke gegenüber."
+ agot_bm_6699:0 "Das Licht der Sieben"
+ agot_bm_6699_desc:0 "Als die Kämpfe weitgehend vorüber waren und die größten Helden alle besiegt waren, erkannten die weisesten der Ersten Menschen, dass es nicht möglich war, der Flut der Invasoren zu trotzen. Viele von ihnen sahen auch ihren Weg, die Verehrung der Sieben Götter des Andal zu übernehmen. Was wird die Zukunft für Westeros in diesem neuen Zeitalter bringen?"
```

Comment:
"Viele von ihnen sahen sich auch in der Lage, die Verehrung der sieben Götter der Andalen zu übernehmen."